src/main/scala/com/atomist/rug/function/github/InstallOrgWebHookFunction.scala

```diff
@@ -27,10 +25,8 @@ class InstallOrgWebHookFunction extends AnnotatedRugFunction
 
     try {
       val ghs = gitHubServices(token, apiUrl)
-      val org = ghs.gitHub.getOrganization(owner)
-      val config = Map("url" -> url, "content_type" -> "json")
-      val gHHook = org.createHook("web", config.asJava, Seq(GHEvent.ALL).asJava, true)
-      val response = mapHook(gHHook)
+      val wh = ghs.createOrganizationWebhook(owner, "web", url, "json", active = true, Events.toArray)
```

Comment:
I'm all for consistency. I'll use dot notation for method calls that are in fact, well methods and keep spaces for operators like `+`, `++` etc